code/solr/searchquerywriters/SolrSearchQueryWriter_Basic.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+class SolrSearchQueryWriter_Basic extends AbstractSearchQueryWriter
```

Comment:
If possible, please try and avoid using underscores in class names (for future proofing against PSR-2)